src/main/rx/cc2500_frsky_shared.h

```diff
@@ -33,6 +33,7 @@
 
 #define MAX_MISSING_PKT 100
 
+#define FRSKY_RX_D8_LENGTH     0x11 + 3
```

Comment:
```suggestion
#define FRSKY_RX_D8_LENGTH     (0x11 + 3)
#define FRSKY_RX_D16FCC_LENGTH (0x1d + 3)
#define FRSKY_RX_D16LBT_LENGTH (0x20 + 3)
#define FRSKY_RX_D16v2_LENGTH  (0x1d + 3)
```

To be sure each individual expression is handled as a unit at all times, I think we should put brackets around them.